### PR TITLE
Create Universities Page

### LINKED ISSUE #16: [HTML-CSS] Create Universities page
Adding the required html and Css for the Universities page ,taking in consideration that the containers for the Universities names are only samples for the JS work

### PR DESCRIPTION
**Pull request by nada-abuzaid**

### CODE REVIEW

**Review by basillubbad — APPROVED** [review]

**Review by Yousef-Dall — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
+    </div>
+    <!-- ./cover-img -->
+    <div class="welcome-texts">
+        <h1 class="main-text">Your journey to uni starts here</h1>
+        <p class="more-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+        </p>
+    </div>
+    <!-- ./welcome-texts -->
+    <button class="start-btn" onclick="location.href='searchPage/search.html'">Let’s Go</button>
 </body>
```
Comment (yaqootturman) [inline index.html]: Why do you have landing page work in your pull request? which is not merged yet

File: universitiesPage/universities.html [hunk 2]
```diff
@@ -1,24 +1,79 @@
...
+        (function() {
+            var css = document.createElement('link');
+            css.href = "https://pro.fontawesome.com/releases/v5.10.0/css/all.css";
+            css.rel = 'stylesheet';
+            css.type = 'text/css';
+            document.getElementsByTagName('head')[0].appendChild(css);
+        })();
+    </script>
+    <link rel="stylesheet " href="universities.css">
+    <title>UNIGUIDE</title>
```
Comment (yaqootturman) [inline universitiesPage/universities.html]: the title should refer to page not to the app

File: universitiesPage/universities.html [hunk 3]
```diff
@@ -1,24 +1,79 @@
...
     </header>
-    <script src="Universities.js"></script>
+
+    <main>
+        <p class="section-title ">Universities</p>
+        <div class="container ">
+            <!--  The containers for the Universities names are only samples for the JS work -->
+            <button>
+                <div class="card">
+                    <i class="far fa-school"></i>
```
Comment (yaqootturman) [inline universitiesPage/universities.html:29]: You need to center the icon

File: universitiesPage/universities.html [hunk 4]
```diff
@@ -1,24 +1,79 @@
...
-        <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;400&display=swap" rel="stylesheet">
-        <title>Document</title>
+
+<head>
+
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script type="text/javascript">
+        (function() {
```
Comment (yaqootturman) [inline universitiesPage/universities.html]: move this for separated js file and add a comment to describe it

**Review by yaqootturman — CHANGES_REQUESTED** [review]

**Review by yaqootturman — CHANGES_REQUESTED** [review]
previous changes

**Review by yaqootturman — APPROVED** [review]

**Pull request merged**